EasyInventory/src/main/java/com/easyinventory/enterprise/controller/HTMLController.java

```diff
@@ -21,7 +21,7 @@
 public class HTMLController {
 
     @Autowired
-    EasyInventoryService easyInventoryService;
+    private EasyInventoryService easyInventoryService;
```

Comment:
Based upon java standards, I opted to add a private access modifier to this variables, as they are only accessed in this class.

https://docs.oracle.com/javase/tutorial/java/javaOO/variables.html